python/delete_unlinked_top_containers.py

```diff
@@ -0,0 +1,62 @@
+import asnake.logging as logging
+from asnake.client import ASnakeClient
+import argparse
+
+logging.setup_logging(filename="archivessnake.log", level="INFO")
+# set label for custom logger - all output will be in archivessnake.log
+logger = logging.get_logger("delete_unlinked_top_containers")
+client = ASnakeClient()
+
+
+def container_is_unlinked(top_container_uri: str) -> bool:
+    top_container = client.get(top_container_uri).json()
+    if len(top_container["collection"]) == 0:
+        return True
+    return False
+
+
+def container_exists(top_container_uri: str) -> bool:
```

Comment:
This seems more appropriate as a general error checker, because it doesn't really check whether the container exists - only that the response contains an error.  There probably are other error cases which don't represent the (non)existence of the container, but also indicate we should not try to delete it.

If all we want is an existence check, we don't need to look inside the payload, but can take advantage of the client basically being a wrapper around `requests`.
```
>>> top_container_uri = "/repositories/2/top_containers/99999999"
>>> response = client.get(top_container_uri)
>>> print(response)
<Response [404]>
>>> print(response.status_code)
404
```

I think there's more value in the generic error checking: we can use a more generic function in multiple scripts (refactoring it into a utility file when needed...).  So I recommend just renaming this function to better reflect its general "there's a problem" purpose.